wgpu-core/src/command/render.rs

```diff
@@ -154,6 +154,8 @@ pub struct RenderPassDescriptor<'a> {
     pub color_attachments: Cow<'a, [RenderPassColorAttachment]>,
     /// The depth and stencil attachment of the render pass, if any.
     pub depth_stencil_attachment: Option<&'a RenderPassDepthStencilAttachment>,
+    /// If this render pass will be rendering to multiple array layers.
+    pub multiview: bool,
```

Comment:
is this actually needed? If I understand correctly, we expect all of the attachments to be layered in this case, and with the same number of layers in them. So this flag becomes redundant - one could just check the actual texture views in the attachments.